bundle/src/test/java/com/adobe/acs/commons/httpcache/store/jcr/impl/writer/BucketNodeFactoryTest.java

```diff
@@ -46,12 +46,6 @@
     @Mock Session session;
     @Mock Node cacheRootNode;
 
-    @Before
-    public void setup() throws Exception{
-        when(session.isLive()).thenReturn(true);
-        when(cacheRootNode.getPath()).thenReturn(MockSettings.VALID_ROOT_PATH);
```

Comment:
Removing this doesn't break the tests?  Really?  Interesting...

Reply:
Mockito 2 is telling you alot about your tests ... :-)